gateway/1.6/modules/ROOT/pages/flex-sizing-guide.adoc

```diff
@@ -1,5 +1,62 @@
-= Flex Gateway Resource Sizing
+= Reviewing Resource Sizing 
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:imagesdir: ../assets/images
\ No newline at end of file
+:imagesdir: ../assets/images
+:table-caption!:
+
+Before getting started with Flex Gateway, you must analyze and choose your configuration to obtain the best performance and auto-scaling rate.
+
+When assigning resources for a node, you must consider:
+
+* Number of APIs
+* Amount of memory the node needs
+* Processing power for the node
+* Number of policies in each API
+* Amount of traffic each node handles
+* Amount of time the node takes to answer the request (latency)
+* Type of traffic the node handles, and within the traffic:
+** The number of requests per second the node handles
+** Size of each request (bytes)
+
+== Choose Appropriate Resources
+The following table guides you to know the resources you must allocate to your node according to your environment:
+
+[options="header"]
+|====
+| Node Size | CPU | RAM 
+| Small | 1-2 cores | 2-4 GB
+| Medium | 2-4 cores | 4-8 GB
+| Large | 8-16 cores | 16-32 GB	
+|====
+
+NOTE: Nodes belonging to a cluster must be of the same size.
+
+== Choose Your Cluster Size
+
+You can choose the appropriate cluster size with the following guiding table:
+
+[options="header"]
+|====
+| Node Size | Number of APIs | Number of Policies | Expected Latency | Expected Throughput | Target Environment
+| Small | < 100 | 4 | < 100 ms | < 500 RPS | Dev/test cluster
+| Medium | < 1000 | 4 | < 20 ms | < 2500 RPS | Production cluster
+| Large | < 10000 | 4 | < 10 ms | < 10000 RPS | Mission-critical clusters
+|====
+
+NOTE: A high CPU can lead to increased latency. Proceed with horizontal scaling or increase the node size. High memory consumption may lead to out-of-memory errors. Increase the memory allocation or the node size. Horizontal scaling can fail helping in high-memory situations.
```

Comment:
A high CPU usage can lead.. << Please reword
Horizontal scaling will fail when high memory utilization is the problem, first increase memory then scale horizontally. << Please reword last sentence

Reply:
OK! Thanks!